Person.rb

```diff
@@ -0,0 +1,11 @@
+
+class Person
+	attr_reader :age 
+	attr_accessor :name
+
+	def initialize(options = {})
```

Comment:
Indentation!  If this looked right locally, you are probably using tabs instead of spaces in sublime, try setting your preferences to 2 spaces:

http://stackoverflow.com/questions/9474090/how-do-i-force-sublime-text-2-to-indent-two-spaces-per-tab